apps/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -1,35 +1,23 @@
-import { Component, Input } from '@angular/core';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Output,
+} from '@angular/core';
 
 @Component({
   selector: 'app-list-item',
   template: `
     <div class="border border-grey-300 py-1 px-2 flex justify-between">
-      {{ name }}
-      <button (click)="delete(id)">
+      <ng-content></ng-content>
+      <button (click)="delete.next()">
```

Comment:
Hey, I have a question, why did you use delete.next() instead of delete.emit(), Ive researched this and nextCall() seems to be deprecated

https://github.com/angular/angular/blob/b5b6ece65a96f5b8f134ad4899b56bf84afe3ba0/modules/angular2/src/facade/async.dart#L49